### ISSUE TITLE
JSHint complains about some things

### ISSUE THREAD

**Issue by vniche** [gallera/blog-entities#3]
Running JSHint on the project structure, i get the following errors:

`jshint src/*`

```
src/password.js: line 27, col 44, Missing semicolon.
src/password.js: line 32, col 65, Missing semicolon.
src/password.js: line 38, col 27, Missing semicolon.

src/profile.js: line 15, col 42, Missing semicolon.
src/profile.js: line 21, col 46, Missing semicolon.
src/profile.js: line 13, col 7, Setter is defined without getter.
src/profile.js: line 19, col 7, Setter is defined without getter.
src/profile.js: line 25, col 7, Setter is defined without getter.

src/user.js: line 19, col 43, Missing semicolon.
src/user.js: line 24, col 39, Missing semicolon.
src/user.js: line 27, col 24, Missing semicolon.
src/user.js: line 31, col 46, Missing semicolon.
src/user.js: line 46, col 45, Missing semicolon.
src/user.js: line 17, col 7, Setter is defined without getter.
src/user.js: line 29, col 7, Setter is defined without getter.
src/user.js: line 44, col 7, Setter is defined without getter.

16 errors
```

`jshint test/*`

```
test/password.js: line 11, col 48, Missing semicolon.
test/password.js: line 16, col 57, Missing semicolon.
test/password.js: line 21, col 61, Missing semicolon.

test/profile.js: line 10, col 47, Missing semicolon.
test/profile.js: line 15, col 56, Missing semicolon.
test/profile.js: line 20, col 72, Missing semicolon.

test/user.js: line 13, col 44, Missing semicolon.
test/user.js: line 18, col 54, Missing semicolon.
test/user.js: line 23, col 58, Missing semicolon.
test/user.js: line 28, col 77, Missing semicolon.
test/user.js: line 34, col 84, Missing semicolon.
test/user.js: line 40, col 95, Missing semicolon.

12 errors
```

I think, a great next move on the project would be to get those things rights, and run it with Travis CI.

**Comment by vniche** [comment]
Used the following `.jshintrc` configuration:

```
{
    "esversion": 6,
    "mocha": true,
    "node": true
}
```

**Comment by vniche** [comment]
Issue resolved with PR #5
